src/app/shared/models/fcmUserDto.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * App Server
+ * RADAR App Server Api Documentation
+ *
+ * OpenAPI spec version: 0.0.1-SNAPSHOT
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+
+export interface FcmUserDto { 
+    enrolmentDate?: Date;
+    fcmToken?: string;
+    id?: number;
+    lastDelivered?: Date;
+    lastOpened?: Date;
+    projectId?: string;
+    subjectId?: string;
+    timezone?: number;
```

Comment:
Here too, make the fields non-optional/null. enrolmentDate, fcmToken, projected, subjected, timezone cannot be null. Moreover, there is a field called `language`